### ISSUE TITLE
TCJA on dvs128

### ISSUE THREAD

**Issue by Jee-King** [ridgerchu/TCJA#1]
Hi, thanks for sharing your great work! TCJA achieve 99.0% acc on dvs128! 
why did you only use TCJA in the last two-layer?  
https://github.com/ridgerchu/TCJA/blob/f7db476172ecd725aee2c78b00ea09d7182d0b91/src/dvs128.py#L40

By the way, could you give me some suggestions about how to train a model on dvs128? As I change the network, the performance will drop a lot. is it sensitive to the parameters?

**Comment by ridgerchu** [comment]
We discovered that if we added the TCJA to all layers, the performance of this network would not considerably improve, and the computational cost would also rise. For this reason, we decided to only add the TCJA to the bottom 2 layers. 
Regarding the dvs128, I believe that the trick to training it is to avoid using too deep network architecture or ResNet, and avoid replacing the last layer from LIF to MLP. This method has been proven to work with other datasets.

**Comment by ridgerchu** [comment]
Hi Jee, I apologize for the delay in responding to your question. 
We discovered that if we added the TCJA to all layers, the performance of this network would not considerably improve, and the computational cost would also rise. For this reason, we decided to only add the TCJA to the bottom 2 layers. 
Regarding the dvs128, I believe that the trick to training it is to avoid using too deep network architecture or ResNet, and avoid replacing the last layer from LIF to MLP. This method has been proven to work with other datasets.
